test/dist_autograd_test.py

```diff
@@ -735,12 +780,12 @@ def test_backward_invalid_args(self):
                 t = torch.rand(1, requires_grad=True)
                 dist_autograd.backward([t])
 
-    @dist_init
+    @dist_init(setup_model_parallel=True)
```

Comment:
Can we add the following tests:

1) Never call RRef.to_here() and verify grads are accumulated on all tensors except the RRef tensor.
2) A creates an RRef on B and shares that RRef with C. C now calls to_here() and does some computation and returns the result to A. A now calls backward on the result. We should verify the OwnerRRef on B accumulates gradients appropriately.
3) Multiple nodes call RRef.to_here() on the same rref and perform some computation. The backward pass should result in multiple edges to the same RRef and appropriate gradient accumulation. We could denote one node as the parameter server for this test and retrieve the RRef to do some computation.